test/dummy/.node-version

```diff
@@ -0,0 +1 @@
+18.15.0
```

Comment:
Is there a reason this isn't `20.x` or `22.x`?

Reply:
Also, should we add a newline at the end of the file?

Reply:
@Braden-077 thanks for catching this.  I believe one of the rolemodel_rails generators added this.  Going to see if it's needed at all..

Reply:
@Braden-077 ended up setting this to 20.12.0 (current lts version) Thanks 👍